translators/tmgmt_local/config/schema/tmgmt_local.schema.yml

```diff
@@ -1,3 +1,7 @@
 tmgmt.translator.settings.local:
   type: mapping
```

Comment:
We do have a base type for this (tmgmt.translator_base), but that is now empty, so doesn't really matter if you use it or not.